src/library-authoring/LibraryAuthoringPage.jsx

```diff
@@ -90,7 +95,6 @@ const LibraryAuthoringPage = () => {
         <SearchField
           value={searchKeywords}
           placeholder={intl.formatMessage(messages.searchPlaceholder)}
-          onSubmit={(value) => setSearchKeywords(value)}
```

Comment:
The `onSubmit` is needed otherwise if you press 'Enter' it will error out: `onSubmit is not a function`. To avoid performing the search again we can just pass in a no-op function `() => {}`.

Reply:
I realized this is the same comment as the one I added below.